app/views/automated_tests/_test_script_upload.html.erb

```diff
@@ -18,6 +18,8 @@
     <span class='file_name'>
       <% if saved %>
         <%= f.object.script_name %>
+        <%= f.hidden_field :script_name, :value => f.object.script_name,
```

Comment:
Use the new hash syntax `value: f.object.script_name ...`. Also use single-quotes. Wait, why is there a class at all?